packages/uif/src/reducer/handlers/handleRetryUpdate.ts

```diff
@@ -0,0 +1,22 @@
+import assert from 'node:assert'
+
+import { continueOperations } from '../helpers/continueOperations'
+import { RetryUpdateAction } from '../types/IndexerAction'
+import { IndexerReducerResult } from '../types/IndexerReducerResult'
+import { IndexerState } from '../types/IndexerState'
+
+export function handleRetryUpdate(
+  state: IndexerState,
+  _action: RetryUpdateAction,
+): IndexerReducerResult {
+  assert(state.retryingUpdate, 'should be retrying update')
+
+  if (state.status === 'invalidating' || state.retryingInvalidate) {
+    return [state, [{ type: 'ScheduleRetryUpdate' }]]
```

Comment:
What is the use case? We scheduled a retry but it was too early? Woudn't we automatically update after invalidate anyway? Do we need this entire if?

Reply:
So the idea here that not, we would not update after we invalidate, because we have the `updateBlocked` flag on. The idea is that if the `invalidate` is super quick, we could end up spamming some flaky API 15 times a second.

Reply:
updateBlocked = retryingUpdate